internal/arch/writer_wiring_test.go

```
@@ -0,0 +1,92 @@
+		if !strings.HasSuffix(path, ".go") {
+			return nil
+		}
+		// Skip worktrees, vendored trees, and generated areas.
+		if strings.Contains(path, "/.claude/") || strings.Contains(path, "/vendor/") {
+			return nil
+		}
+		// Tests neither declare production writers nor count as production callers.
+		if strings.HasSuffix(path, "_test.go") {
+			return nil
+		}
+		src, readErr := os.ReadFile(path)
```

golangci-lint (2.12.2)

[medium] G304: Potential file inclusion via variable

(gosec)